dts/bindings/sensor/nxp,fxos8700.yaml

```diff
@@ -31,3 +31,89 @@ properties:
         This pin defaults to active low when produced by the sensor.
         The property value should ensure the flags properly describe
         the signal that is presented to the driver.
+
+    range:
+      type: int
+      required: false
+      description: Range in g
+      enum:
+        - 8 # 8g (0.976 mg/LSB)
+        - 4 # 4g (0.488 mg/LSB)
+        - 2 # 2g (0.244 mg/LSB)
+
+    power-mode:
+      type: int
+      required: false
+      description: Power mode
+      enum:
+        - 0 # Normal
+        - 1 # Low noise, low power
+        - 2 # High resolution
+        - 3 # Low power
```

Comment:
It would probably be good to also introduce some device tree header with definitions for that.